contributingGuides/STYLE.md

```diff
@@ -210,30 +210,28 @@ const {name, accountID, email} = data;
 
 **React Components**
 
-Don't destructure props or state. It makes the source of a given variable unclear. This guideline helps us quickly know which variables are from props, state, or from some other scope.
+Always use destructing to get prop values. Destructing is necessary to assign default values to props. 
```

Comment:
```suggestion
Always use destructuring to get prop values. Destructuring is necessary to assign default values to props. 
```

Reply:
Can you add some explanation for why it is necssary? 

If we didn't have `defaultProps` then I think the alternative would be something like:

```js
const propWithDefault = props.something || 'default';
```

Reply:
@marcaaron 

did you mean

```js
function MyComponent({ propWithDefault = "default" }) {...};
```

Reply:
by the way, I like "destructing" 🦖 😂